src-tauri/src/manager.rs

```diff
@@ -53,11 +56,47 @@ impl ManagerState {
         self.watchers_running.insert(name.to_string(), true);
         self.watchers_pid.insert(name.to_string(), pid);
         println!("{:?}", self.watchers_running);
+        self.update_tray_menu();
     }
     fn stopped_watcher(&mut self, name: &str) {
         println!("stopped {name}");
         self.watchers_running.insert(name.to_string(), false);
         self.watchers_pid.remove(name);
+        self.update_tray_menu();
```

Comment:
What if a watcher crashes? Then this wouldn't run.

Reply:
I have manually killed the watchers during testing it does run. The monitoring of the watchers is done by the thread and immediately sends the message on watcher crash.

Reply:
Ah right, forgot about the function not being used to stop a watcher, but to notify a watcher has stopped.